lib/modules/datasource/repology/index.ts

```diff
@@ -177,7 +177,7 @@ export class RepologyDatasource extends Datasource {
       throw err;
     }
 
-    logger.debug(
```

Comment:
@rarkins I think we should throw the error up instead of catching here

Reply:
To be caught by the datasource index function?

Reply:
ok, throwing the Error. Sounds good to mw.

Reply:
> To be caught by the datasource index function?

yes